style.css

```diff
@@ -674,3 +691,160 @@
   order: 0;
   flex-grow: 0;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 375px;
+  height: auto;
+  left: 0;
+  top: 6238px;
+}
+
+.contact-intro-cont {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 216px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.contact-intro-cont p {
+  width: 327px;
+  height: 216px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.the-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: auto;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  text-align: center;
+}
+
+.name-div {
+  height: 48px;
+  left: 0;
+  right: 0;
+  top: calc(50% - 48px / 2);
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+#name {
+  height: 18px;
+  left: 16px;
+  right: 16px;
+  top: calc(50% - 18px / 2);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+}
+
+.email-div {
+  height: 48px;
+  left: 0;
+  right: 0;
+  top: calc(50% - 48px / 2);
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+#email {
+  height: 18px;
+  left: 16px;
+  right: 16px;
+  top: calc(50% - 18px / 2);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+}
+
+.message-div {
+  width: 327px;
+  height: 114px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+#message {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+.get-in-touch {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 131px;
+  height: 48px;
+  background: #ff6b00;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  border: none;
+  color: #fff;
+  margin-top: 26px;
+}
+
+.socials-footer {
+  display: flex;
+  order: 4;
+  list-style: none;
+}
+
+.socials-footer li {
+  margin-left: 20px;
+}

```

Comment:
- I think your social icons and the `<hr>` break line are not properly positioned. Also, there is excess space between all the social icons and the hr break line. Can you kindly fix this?

Here is what I could see compared to what's on Figma 👇🏽 

| After clicking  Icon on Your Design  |After clicking  Icon on Figma Design |
| :---:   | :---: | 
|![bdicon](https://user-images.githubusercontent.com/99246359/200667302-8d5bc150-8e88-4d15-8cb0-d21aa5bb4a93.PNG)  |![gdicon](https://user-images.githubusercontent.com/99246359/200667442-531dce66-f9bb-4b8b-83c0-fceaffc465e2.PNG)  |  



[OPTIONAL] 👇🏽 

- I see that you have used downloaded images for your icons and this may have caused some default dull color on each of them. Apparently, It would be impossible to assign a color to the icons.
If you would be able, I will like to suggest you consider replacing the images with font awesome icons because it will be easier to tweak the colors as you like. This way, the colors of your icons will match that on Figma template. 🤗